### PR TITLE
Removed references to CIMPL V5

### PR DESCRIPTION
**Pull request by GHCouderc**
Made minor changes documented in SHRO-870, and from cc review.

### CODE REVIEW

File: docs/cimpl6UserGuide.md [hunk 1]
```diff
@@ -69,35 +69,35 @@ As a best practice, CIMPL map files must have the *_map.txt* naming convention.
...
 Where:
 
-* _namespace name_ is the name of the grouping of elements that you have identified.
+* _data element group name_ is the name of the grouping of elements that you have identified.
 * _fhir target version_ is a choice of FHIR_DSTU_2, FHIR_STU_3, or FHIR_R4
 
 The snippet below shows an example:
 ```
-Grammar:	Map 5.1
+Grammar:	Map 6.0
```
Comment (mlterryMitre) [inline docs/cimpl6UserGuide.md]: @GHCouderc - actually, it's Map 5.1, not 6.0.

File: docs/cimpl6UserGuide.md [hunk 2]
```diff
@@ -133,7 +140,15 @@ CIMPL Compilation Errors are structured in the following format:
...
 A detailed list of CIMPL compilation errors and troubleshooting suggestions are available **[here](https://github.com/standardhealth/shr-cli/wiki/Error-Message-Documentation)**.
 
-## IG Publisher Output
+## FHIR IG Publisher Output
+
+The final step in the IG creation process is to run the **[FHIR IG Publisher](http://wiki.hl7.org/index.php?title=IG_Publisher_Documentation)**. This tool is maintained and owned by HL7 FHIR. 
+
+At a command prompt, enter (changing mcodeR4 to your CIMPL model name, and the -ig name to an appropriate value):  (Mark & May)
+
+* yarn run ig: publish   QUESTION: readme has this - this needs work
```
Comment (mlterryMitre) [inline docs/cimpl6UserGuide.md]: yarn run ig: publish is used if you have taken the defaults for running `shr-cli` (which I think puts the output into the same directory as the source code.  The more extensive IG Publisher command allows you to generate the IG output from any directory.

File: docs/cimplInstall.md [hunk 3]
```diff
@@ -34,6 +34,7 @@ setx https_proxy https://your.proxy.org
 setx proxy http://your.proxy.org  
 setx JAVA_OPTS "-Dhttp.proxyHost=your.proxy.org -Dhttp.proxyPort=80 -Dhttps.proxyHost=your.proxy.org -Dhttps.proxyPort=80 -DsocksProxyHost=your.proxy.org -DsocksProxyPort=80"
```
Comment (mlterryMitre) [inline docs/cimplInstall.md]: I don't know that we need the JAVA_OPTS environment setting anymore.  cc: @cmoesel to weigh in.

File: docs/cimplInstall.md [hunk 4]
```diff
@@ -333,7 +353,7 @@ gem install bundler jekyll
 
 _NOTE: If your system requires `sudo` to gem install bundler and jekyll, use the following command to preserve environment variables when using `sudo`:_
 ```
-sudo -E gem install bundler jekyl
+sudo -E gem install bundler jekyll
```
Comment (mlterryMitre) [inline docs/cimplInstall.md]: good catch on this typo! :-)

File: docs/cimplInstall.md (hunk 3 above)
Comment (cmoesel) [reply docs/cimplInstall.md]: It's still needed if you're in an environment w/ an explicit proxy.  At MITRE the macs have been transitioned to transparent proxy, so we don't need it anymore -- but windows are still using the explicit proxy.

All that said, last time I checked, IG Publisher wasn't honoring the Java proxy settings anyway. :-/

File: docs/cimplInstall.md (hunk 4 above)
Comment (GHCouderc) [reply docs/cimplInstall.md]: Mint caught that one!

File: docs/cimpl6UserGuide.md (hunk 1 above)
Comment (GHCouderc) [reply docs/cimpl6UserGuide.md]: ok

File: docs/cimplInstall.md (hunk 3 above)
Comment (GHCouderc) [reply docs/cimplInstall.md]: Do I leave it in the install documentation or not?  I ran this command today without JAVA_OPTS in a windows environment and it worked although the output I get isn't as expected.

**Pull request merged**